README.md

```diff
@@ -49,58 +51,76 @@ git clone https://github.com/o3de/o3de.git
     ```
-    > Note:  Do not use trailing slashes for the <3rdParty cache path>
+    
+    > Note:  Do not use trailing slashes for the <3rdParty cache path>.
 
 1.  Alternatively, you can do this through the CMake GUI:
     
-    1.  Start `cmake-gui.exe`
-    1.  Select the local path of the repo under "Where is the source code"
-    1.  Select a path where to build binaries under "Where to build the binaries"
-    1.  Click "Configure"
-    1.  Wait for the key values to populate. Fill in the fields that are relevant, including `LY_3RDPARTY_PATH` and `LY_PROJECTS`
-    1.  Click "Generate"
+    1.  Start `cmake-gui.exe`.
+    1.  Select the local path of the repo under "Where is the source code".
+    1.  Select a path where to build binaries under "Where to build the binaries".
+    1.  Click **Configure**.
+    1.  Wait for the key values to populate. Fill in the fields that are relevant, including `LY_3RDPARTY_PATH`.
+    1.  Click **Generate**.
     
-1.  The configuration of the solution is complete. To build the Editor and AssetProcessor to binaries, run this command inside your repo:
+1.  Register the engine with this command:
     ```
-    cmake --build <your build path> --target AutomatedTesting.GameLauncher AssetProcessor Editor --config profile -- /m
+    scripts\o3de.bat register --this-engine
     ```
-   
-1.  This will compile after some time and binaries will be available in the build path you've specified
 
-### Setting up new projects
-1.  While still within the repo folder, register the engine with this command:
-    ```
-    scripts\o3de.bat register --this-engine
+1.  The configuration of the solution is complete. You are now ready to create a project and build the engine.
+
+For more details on the steps above, refer to [Setting up O3DE from GitHub](https://o3de.org/docs/welcome-guide/setup/setup-from-github/) in the documentation.
+
+### Setting up new projects and building the engine
+
+1. From the O3DE repo folder, set up a new project using the `o3de create-project` command.
     ```
-1. Setup new projects using the `o3de create-project` command.
+    scripts\o3de.bat create-project --project-path <your new project path>
     ```
-    <Repo path>\scripts\o3de.bat create-project --project-path <your new project path>
+
+1. Configure a solution for your project.
     ```
-1. Register the engine to the project
+    cmake -B <your project build path> -S <your new project source path> -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<3rdParty cache path> -DLY_UNITY_BUILD=ON
     ```
-    <Repo path>\scripts\o3de.bat register --project-path <New project path>
+
+    Example:
     ```
-1.  Once you're ready to build the project, run the same set of commands to configure and build:
+    cmake -B C:\my-project\build\windows_vs2019 -S C:\my-project -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_UNITY_BUILD=ON
```

Comment:
repeat: `-DLY_UNITY_BUILD=ON` can be omitted here to simplify the lines

Reply:
Done.